tests/unit/test-class-wc-payment-gateway-wcpay-subscriptions-process-payment.php

```diff
@@ -392,20 +406,16 @@ function ( $order ) {
 			->expects( $this->never() )
 			->method( 'add_payment_method_to_user' );
 
-		// Expect add token to order to be called, so it can be reused in renewals.
-		$this->mock_wcpay_gateway
-			->expects( $this->once() )
-			->method( 'add_token_to_order' )
-			->with(
-				$this->callback( $this->match_order_id( $order->get_id() ) ),
-				$this->token
-			);
-
 		$result       = $this->mock_wcpay_gateway->process_payment( $order->get_id() );
 		$result_order = wc_get_order( $order->get_id() );
 
 		$this->assertEquals( 'processing', $result_order->get_status() );
 		$this->assertEquals( 'success', $result['result'] );
+		// Expect add token to order to be called, so it can be reused in renewals.
+		// This is an integration test, different scenarios for add_token_to_order method
+		// are tested in WC_Payment_Gateway_WCPay_Subscriptions_Test.
+		$payment_tokens = $order->get_payment_tokens();
```

Comment:
Nitpick: this two-line check is used quite often in the tests. Maybe it can be extracted into the the helper function, e.g. `assertOrderPaymentTokenEquals`? And another one similar to it for subscriptions `assertOrderSubscriptionsPaymentTokenEquals` or something like that?